actionview/lib/action_view/helpers/atom_feed_helper.rb

```diff
@@ -191,7 +191,9 @@ def entry(record, options = {})
 
             type = options.fetch(:type, 'text/html')
 
-            @xml.link(:rel => 'alternate', :type => type, :href => options[:url] || @view.polymorphic_url(record))
+            unless options[:url] == false
+              @xml.link(:rel => 'alternate', :type => type, :href => options[:url] || @view.polymorphic_url(record))
+            end
```

Comment:
I'd do something like this ...

```
            url = options.fetch(:url) { @view.polymorphic_url(record) }
            @xml.link(:rel => 'alternate', :type => type, :href => url) if url
```

Because you want @view.polymorphic_url(record) to be executed if nobody set options[:url] if somebody does, regardless if it's nil or false you want to skip that.